app/src/main/res/layout/fragment_event_detail.xml

```diff
@@ -36,8 +36,8 @@
                 android:layout_gravity="bottom"
                 android:textColor="@color/white"
                 android:textSize="@dimen/text_xxlarge"
-                android:paddingLeft="@dimen/activity_horizontal_margin"
                 android:paddingStart="@dimen/activity_horizontal_margin"
+                android:paddingEnd="0dp"
```

Comment:
Is there any reason why we're explicitly putting paddingEnd/Start as 0 instead of just nothing there?